pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -301,17 +301,22 @@ func (p *provider) Create(machine *v1alpha1.Machine, userdata string) (instance.
 		}
 	}
 
-	err = ensureKubernetesSecurityGroupExist(client, c.Region, securityGroupName)
-	if err != nil {
-		return nil, fmt.Errorf("failed to ensure that the kubernetes security group %q exists: %v", securityGroupName, err)
+	securityGroups := c.SecurityGroups
+	if len(c.SecurityGroups) == 0 {
```

Comment:
This doesn't make any actual difference, but wouldn't it it be more consistent to only use `securityGroups` instead of `c.SecurityGroups` after assigning the latter to the former?

Reply:
good point!